databricks/koalas/series.py

```diff
@@ -189,7 +190,7 @@ def spark_type(self):
         return self.schema.fields[-1].dataType
 
     def astype(self, dtype):
-        from databricks.koalas.typing import as_spark_type
+        from databricks.koalas.typehint import as_spark_type
```

Comment:
`typing` -> `typehint` was a workaround for, something I suspect a bug in PyCharm:

```
Traceback (most recent call last):
  File "/.../pycharm/docrunner.py", line 297, in <module>
    modules = [loadSource(a[0])]
  File "/.../pycharm/docrunner.py", line 229, in loadSource
    module = imp.load_source(moduleName, fileName)
  File "/.../lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/.../koalas/databricks/koalas/series.py", line 24, in <module>
    import pandas as pd
  File "/.../lib/python3.7/site-packages/pandas/__init__.py", line 23, in <module>
    from pandas.compat.numpy import *
  File "/.../lib/python3.7/site-packages/pandas/compat/__init__.py", line 446, in <module>
    import typing
  File "/.../koalas/databricks/koalas/typing.py", line 33, in <module>
    T = typing.TypeVar("T")
AttributeError: module 'typing' has no attribute 'TypeVar'
```
Looks executing doctests fails to resolve namespace propery and it looks getting confused by, current module's `typing` and Python built-in's `typing`.